### PR TITLE
healthlink card added

### PR DESCRIPTION
**Pull request by mk08sh**
## Status
Ready

## Fixes or Implements [AB#7861](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7861)
- [x] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
A clickable card with healthlink info that links to healthlink website for additional information to the user

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [ x] Not Required

### Steps to Reproduce:
If this is a bug, please provide details on how to reproduce the code.

### UI Changes
YES 

### Browsers Tested
- [x] Chrome
- [x] Safari
- [ ] Edge
- [ ] Firefox

Stephen will perform functional testing [AB#7864](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7864)

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+@Component
+export default class HealthlinkSidebarComponent extends Vue {
+  private currentCard: Healthcard = { title: "", description: "", url: "" };
+  private cardPool: Healthcard[] = [
+    {
+      title: "8-1-1",
+      description:
+        "Speak to a pharmacist from 5pm to 9am Pacific Time everyday of the year.",
+      url: "sdsa.com"
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: Not real links?

File: Apps/WebClient/src/appsettings.Development.json [hunk 2]
```diff
@@ -1,60 +1,60 @@
 ﻿{
-    "Logging": {
```
Comment (FuriousLlama) [inline Apps/WebClient/src/appsettings.Development.json]: file formating is off

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      title: "Healthlink",
+      description:
+        "HealthLink BC provides reliable non-emergency health information and advice in British Columbia.",
+      url: "lab.com"
+    }
+  ];
+  mounted() {
+    let cardIndex: number = Math.floor(Math.random() * this.cardPool.length);
+    this.currentCard = this.cardPool[cardIndex];
+    console.log(this.currentCard);
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: console log not necessary

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      target="_blank"
+      class="stretched-link"
+    ></a>
+    <b-card-text>{{ currentCard.description }}</b-card-text>
+  </b-card>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Prop, Component } from "vue-property-decorator";
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: Prop not used below

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  pointer-events: auto;
+  content: "";
+  background-color: rgba(0, 0, 0, 0);
+}
+</style>
+
+<template>
+  <b-card
+    :title="currentCard.title"
+    img-top
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: this does not look like a valid attribute

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    tag="article"
+    style="max-width: 50rem;border: none;"
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: style should be moved to the css section or better, use bootstrap defaults

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+  margin: 100px auto;
+  width: 160px;
+  background: white;
+  border-radius: 5px;
+}
+.box:hover {
+  background: rgba(0, 0, 0, 0.022);
+}
+.stretched-link::after {
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: not sure how these styles are used. Can we use bootstrap defaults instead?

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue [hunk 8]
```diff
@@ -0,0 +1,88 @@
+<style lang="scss" scoped>
+.box {
+  box-shadow: 0 2.8px 2.2px rgba(0, 0, 0, 0.015),
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: bootstrap has shadow classes, lets use those instead
https://getbootstrap.com/docs/4.4/utilities/shadows/

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue (hunk 1 above)
Comment (mk08sh) [reply Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: removed the links, the link will be part of the b-card

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue (hunk 3 above)
Comment (mk08sh) [reply Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: removed

File: Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue (hunk 8 above)
Comment (mk08sh) [reply Apps/WebClient/src/ClientApp/app/components/timeline/healthlink.vue]: revised to bootstrap default

File: Apps/WebClient/src/appsettings.Development.json (hunk 2 above)
Comment (mk08sh) [reply Apps/WebClient/src/appsettings.Development.json]: i think i fixed it

Comment (FuriousLlama) [reply Apps/WebClient/src/appsettings.Development.json]: not fixed

**Pull request merged**